### ISSUE TITLE
Some highway=path is not shown if OpenGL rendering is selected

### ISSUE THREAD

**Issue by Ircama** [osmandapp/Osmand#4696]
For some (strange) reason, some *highway=path* like [this one which includes *highway=path, ref=327, sac_scale=mountain_hiking*]( https://www.openstreetmap.org/way/515423698#map=18/45.94844/10.03632) is correctly shown with the legacy rendering engine, but only rendered as text label (e.g., just the *ref*) and not for its polyline, if the OpenGL rendering engine is selected.

**Comment by xmd5a2** [comment]
Can't reproduce
![screenshot from 2017-11-20 21-59-11](https://user-images.githubusercontent.com/5829014/33036133-a18e895a-ce3e-11e7-82f9-dbe5e54ffc01.png)

**Comment by Ircama** [comment]
@xmd5a2, your sample looks like *OsmAnd* default style (at z18). What I'm using is *Touring view (contrast and details)* style. I did not realize that the issue is just related to this style.

**Comment by xmd5a2** [comment]
@sonora is maintainer of Touring view style

**Comment by sonora** [comment]
Sorry, I had missed this issue completely. Leonid, any idea why this could be? I am aware that Touring view uses our original render syntax, but can this be an issue at all?

**Comment by xmd5a2** [comment]
I think this is related to OpenGL engine inheritance rules

**Comment by sonora** [comment]
So it sounds like nothing to fix in our render.xml's, instead it affects all our dependent renderers and needs to be fixed in our GL engine?

**Comment by vshcherb** [comment]
Yes... Unfortunately. I put a tag OpenGL, there is a chance we got back to those issues after IOS will mature 1-2 more levels.
